gnd/src/main/java/com/google/android/gnd/model/Mutation.java

```diff
@@ -86,7 +91,9 @@ public String toString() {
 
     public abstract T setProjectId(String newProjectId);
 
-    public abstract T setUserId(String newUserId);
+    public abstract T setUserId(@Nullable String newUserId);
```

Comment:
Instead of passing userId, can we pass `User` object instead? The reason is so that we don't have to load `User` again from the id in `RoomLocalDataStore`
See usages of `loadUser` in `RoomLocalDataStore`

Reply:
But that would require adding `*MutationEntityRelations` to fetch `User` when loading from local db.

Reply:
That would push the loading of the `User` to the time of loading `Mutations`, which at DB level would mean either loading each `User` individually, or doing a join to fetch them together. Loading them individually isn't a big deal since they're small, there will be few of them (max 2?) and the db is practically in memory. If we go that route, I think doing a query to fill them in the persistence later is more discoverable than using `MutationEntityRelations`. 

Filed #301 to revisit this.